packages/ui/src/composables/useLoading.ts

```diff
@@ -11,13 +9,13 @@ export type LoadingProps = {
  * It's better to add props at the beginning, to make sure that Component own props will be used instead in case of collision
  */
 export const useLoadingProps = {
-  loading: { type: Boolean as PropType<boolean>, default: false },
+  loading: { type: Boolean, default: false },
 }
 
-export const useLoading = () => {
-  const hasLoading = true
+// export const useLoading = () => {
+//   const hasLoading = true
 
-  return {
-    hasLoading,
-  }
-}
+//   return {
+//     hasLoading,
+//   }
+// }
```

Comment:
If someone uses it already, and we want to remove this method, maybe we need to throw the warning?

Reply:
I checked: this hook is not used anywhere